lib/oli_web/router.ex

```diff
@@ -379,7 +379,6 @@ defmodule OliWeb.Router do
 
     # Editors
     get("/:project_id/resource/:revision_slug", ResourceController, :edit)
-    get("/:project_id/resource/:revision_slug/activity/:activity_slug", ActivityController, :edit)
```

Comment:
why is this removed?

Reply:
The /js/activityeditor.js app it used was removed 9 months ago from the build, effectively breaking this page. I'm making the (hopefully reasonable) assumption that none of this is needed anymore and cleaning the template, view, and controller up.

See commit 52e906613add34ab859c1e35dff40fe9a8d350aa